content/exchange/artifacts/Exchange.Windows.System.PowerShell.DetectResponder

```diff
@@ -0,0 +1,34 @@
+name: Custom.Windows.System.PowerShell.DetectResponder
```

Comment:
Dont name the artifact with custom in the exchange

Reply:
I changed the name of it to Exchange.